vllm/model_executor/models/idefics3.py

```diff
@@ -72,16 +71,38 @@ class Idefics3ImageEmbeddingInputs(TypedDict):
     """
 
 
+class Idefics3ProcessorSize(NamedTuple):
```

Comment:
I think a `TypedDict` would be more appropriate for this?

Reply:
`TypeDict` is unhashable and will raise `TypeError: unhashable type: 'dict'` when using with `lru_cache`.

However, `NamedTuple` is hashable, and we just need to override the `__getitem__` and `__contains__` to make it work like a dict.

Reply:
I see, thanks for the explanation!

Reply:
Can you add a note in the code for this?

Reply:
Added.